src/applications/mhv-secure-messaging/containers/Compose.jsx

```diff
@@ -35,8 +34,14 @@ const Compose = () => {
       } else {
         dispatch(retrieveMessageThread(draftId));
       }
+
+      const checkNextPath = history.listen(nextPath => {
```

Comment:
do we need to address this linter warning and add `history` to dependencies for this `useEffect`?